### ISSUE TITLE
2.4.11 Editor's note wording

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#2006]
Current text:
> The working group is interested in feedback about the minimum area metric, and if there are unusual scenarios where visible indicators are caught by the wording.

Should this be:

> The working group is interested in feedback about the minimum area metric, and if there are unusual scenarios where visible indicators are **not** caught by the wording.

**Comment by alastc** [comment]
Hi Detlev,

Originally I'd put that in to mean: If a good indicator is failed by the wording, i.e. a false positive. So for that purpose I think it's correct.

The opposite would be if "**non-**visible indicators are not caught".

Also, it's an editors note which will be removed before publication, so I've removed the 'errata' tag.
